src/ontodia/customization/defaultLinkStyles.ts

```diff
@@ -1,4 +1,8 @@
-import { LinkTemplate, LinkTemplateResolver } from './props';
+import { LinkTemplate, LinkTemplateResolver, LinkLabel } from './props';
+
+const LINK_LABEL: LinkLabel = {
+    attrs: {text: {'font-weight': 700}},
```

Comment:
Why do you changing the defaults only for sbuClass / domain / range / typeOf link types?